public/locales/en.json

```diff
@@ -189,8 +189,8 @@
                 "description": "Sorry, only the owner can edit this note."
             },
             "limitReached": {
-                "title": "Reach the limit",
-                "description": "Sorry, you've reached the maximum length this note can be.",
+                "title": "Reached the limit",
+                "description": "Sorry, you've reached the maximum length this note can be ({{maxLength}} characters).",
                 "advice": "Please shorten the note."
```

Comment:
```suggestion
                "advice": "Please shorten the note by removing or splitting content into other notes."
```

Reply:
The advice is meant to be a bold, short statement as shown in the screenshot below.
It delivers a short but precise answer what to do to avoid further errors of this kind. I wouldn't say it's necessary to add your suggested explanation.

![image](https://user-images.githubusercontent.com/52606896/92179955-1ecb9000-ee46-11ea-9ba3-cca8cc1a3e9c.png)